samples/bluetooth/unicast_audio_client/src/main.c

```diff
@@ -157,22 +158,27 @@ static void lc3_audio_timer_timeout(struct k_work *work)
 		}
 
 		for (size_t i = 0U; i < configured_sink_stream_count; i++) {
+			struct net_buf *buf_to_send;
 			int ret;
 
-			ret = bt_audio_stream_send(&streams[i], net_buf_ref(buf));
+			/* Clone the buffer if sending on more than 1 stream */
+			if (i == configured_sink_stream_count - 1) {
+				buf_to_send = buf;
+			} else {
+				buf_to_send = net_buf_clone(buf, K_FOREVER);
```

Comment:
Nice - dual mono!

Reply:
:D 

We can always improve on this in the future if we want to send separate data to each side :)